testing/tests/DevExpress.ui.widgets/fileManagerParts/adaptivity.tests.js

```diff
@@ -80,4 +88,24 @@ QUnit.module("Adaptivity", moduleConfig, () => {
         assert.ok(folders.length > 3, "dirs tree visible");
     });
 
+    test("dialog size corrent on different window size", function(assert) {
+        this.wrapper.getToolbarButton("Copy").trigger("dxclick");
+        this.clock.tick(400);
+
+        let $dialog = $(".dx-filemanager-dialog-folder-chooser:visible");
+        assert.equal($dialog.length, 1, "dialog is shown");
+
+        const dialogWidth = $dialog.get(0).offsetWidth;
```

Comment:
Better way to use the $dialog.offset().width construction

Reply:
This approach will not work, because we redefine `jQuery.width()` function (`renderer.fn.width`). Therefore, we can use only the native functions of the DOM element.